src/nora/components/Colors.css

```diff
@@ -1,3 +1,12 @@
 :root {
   --NoraBlue: #1a7ff6;
+
+  --GreenPrimary--opaque: #2bd04b;
```

Comment:
INFO: I wonder if we should make these consistent with `NoraBlue` in terms of a naming convention? I realize the "Nora" part is a bit clumsy, but my thought was it would prevent folks from inadvertently using these outside of Nora.